torch/_dynamo/variables/builtin.py

```diff
@@ -1109,10 +1109,31 @@ def call_setattr(
                 f"setattr(UserDefinedObjectVariable) {type(obj.value).__setattr__}"
             )
         elif isinstance(obj, variables.NNModuleVariable):
+            if isinstance(name_var, ConstantVariable) and obj.is_buffer(tx, name_var):
+                # Its a buffer, so we can mutate the buffer in the graph module itself
+                # TODO - One concern here is that this makes graph module
+                # stateful. There are efforts to make the graph completely
+                # stateless. Conceptually, this buffer mutation can conflict w/
+                # that.
+                name = name_var.value
+                buffer_node = obj.var_getattr(tx, name).as_proxy().node
+                new_value_node = val.as_proxy().node
+                tx.output.create_node(
+                    "call_function", torch.fill_, (buffer_node, new_value_node), {}
+                )
```

Comment:
why fill when the user invoked setattr?

Reply:
Its more like 

```
self.iterations = some_value
```

This some_value is the value that I want to directly put in buffer. I could only find `fill_` op to do that job. Is there anything else that you would prefer?

Reply:
that would still invoke setattr right?